lambdas/jwtauthorizationrequest/src/main/java/uk/gov/di/ipv/cri/passport/jwtauthorizationrequest/JwtAuthorizationRequestHandler.java

```diff
@@ -80,6 +82,13 @@ public APIGatewayProxyResponseEvent handleRequest(
             JarResponse response = generateJarResponse(claimsSet);
 
             return ApiGatewayResponseGenerator.proxyJsonResponse(OK, response);
+        } catch (RecoverableJarValidationException e) {
+            LOGGER.error("JAR validation failed: {}", e.getErrorObject().getDescription());
+            RedirectErrorResponse errorResponse =
+                    new RedirectErrorResponse(
+                            e.getRedirectUri(), e.getErrorObject().toJSONObject());
```

Comment:
I'm not sure if you'd need this toJSONObject method on the RedirectErrorResponse class. We already have a objectMapper write to JSON string thing on the ApiGatewayResponseGenerator.proxyJsonResponse method that would do it I think?

Reply:
Good spot hadn't noticed that 👍 removed the toJSON method now (it's not this one it was the one on line 91)